components/developer/openjade/patches/openjade-05-static-iid.patch

```diff
@@ -0,0 +1,73 @@
+Symbols of static variables are deleted from object files and anyway can be replaced by plain static inline calls
```

Comment:
Is it a known issue?

Reply:
I have not seen anyting like that but other distros maintain their own fork...
Probably a peculiarity of our linker, at least for the virtual function it is clear.
In any case the use of static members is ugly.

Reply:
If you are happy with it, feel free to merge.